libvirt/tests/src/cpu/wrong_topology.py

```
@@ -0,0 +1,51 @@
+    err_msg = params.get('err_msg')
+
+    try:
+        tmp_vmxml = vmxml.copy()
+        tmp_vmxml.setup_attrs(cpu=cpu_attrs)
+        tmp_vmxml.vcpu = vcpu
+
+        tmp_vmxml.vm_name = tmp_vm_name
+        tmp_vmxml.del_uuid()
+        osxml = tmp_vmxml.os
+        os_attrs = osxml.fetch_attrs()
+        for k, v in list(os_attrs.items()):
```

Ruff (0.11.9)

Loop control variable `v` not used within loop body

Rename unused `v` to `_v`

(B007)